nbextensions/usability/toc2/main.js

```diff
@@ -146,26 +147,14 @@ define(["require", "jquery", "base/js/namespace",  'services/config',
     $([IPython.events]).on("notebook_loaded.Notebook", function(){ // curiously, the event is not always fired or detected
```

Comment:
Yes, thanks. I have realized that in the meantime. 
Similarly, the kernel.ready event can also happen before the extension is fully loaded. In the last version, therefore, I load the stuff directly and only update things if these events occur. 
@janschulz I will upload an update tomorrow for the hidden scrollbar issue.